app/src/main/res/layout/fragment_subscriptions.xml

```diff
@@ -24,13 +24,14 @@
             android:id="@+id/feeds_filtered_message"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:paddingStart="60dp"
+            android:paddingHorizontal="16dp"
             android:paddingEnd="0dp"
             android:paddingVertical="4dp"
             android:layout_marginTop="-12dp"
             android:background="?android:attr/selectableItemBackground"
             android:gravity="center_vertical"
-            android:orientation="horizontal">
+            android:orientation="horizontal"
+            tools:ignore="RtlSymmetry">
```

Comment:
This is no longer needed

Reply:
It works but this change is no longer needed. No need to suppress warnings that are no longer there

Reply:
Now the orientation attribute is missing

Reply:
I have resolved the issue